src/App.vue

```diff
@@ -8,11 +8,17 @@ import { RouterView } from 'vue-router';
 
 <style>
 html {
-  height: 100;
+  height: 100%;
+  width: 100%;
+  padding: 0px;
+  margin: 0px;
```

Comment:
Looks like this still isn't filling out the page.

![Screen Shot 2022-06-28 at 9 33 03 AM](https://user-images.githubusercontent.com/80695157/176191677-06b20f02-4f06-4289-ae90-bc97531bdc33.png)